core/src/main/java/com/microsoft/applicationinsights/internal/logger/InternalLogger.java

```diff
@@ -197,6 +207,21 @@ public void trace(String message, Object... args) {
     }
 
     /**
+     * The method will log messages using the Console Logger. Note only messages that are at least INFO will be printed
+     * The method is needed for publishing messages when the logger initialization data is unknown
+     * Either, before reading the file, or when there was error reading that configuration file
+     * @param requestLevel - The level of the message
+     * @param message - The message to print
+     * @param args - The arguments that are part of the message
+     */
+    public void logConfig(LoggingLevel requestLevel, String message, Object... args) {
+        if (configMode) {
+            if (requestLevel.getValue() >= LoggingLevel.INFO.getValue()) {
+                new ConsoleLoggerOutput().log(createMessage(requestLevel.toString(), message, args));
+            }
```

Comment:
Isn't is more simple that 'config' will be a logging level such as that comes before info, warning, error?